src/wp-content/themes/twentytwentyone/inc/template-tags.php

```diff
@@ -219,6 +219,10 @@ function twenty_twenty_one_post_thumbnail() {
 	 * @return void
 	 */
 	function twenty_twenty_one_the_posts_navigation() {
+		$order         = get_query_var( 'order', 'DESC' );
+		$nav_prev_text = ( 'DESC' === $order ) ? __( 'Newer', 'twentytwentyone' ) : __( 'Older', 'twentytwentyone' );
+		$nav_next_text = ( 'DESC' === $order ) ? __( 'Older', 'twentytwentyone' ) : __( 'Newer', 'twentytwentyone' );
```

Comment:
```suggestion
		$order          = get_query_var( 'order', 'DESC' );
		$new_posts_text = __( 'Newer <span class="nav-short">posts</span>', 'twentytwentyone' );
		$old_posts_text = __( 'Older <span class="nav-short">posts</span>', 'twentytwentyone' );
```

Keep existing strings